runtime/src/accounts.rs

```diff
@@ -239,17 +260,26 @@ impl Accounts {
         ancestors: &HashMap<Fork, usize>,
         txs: &[Transaction],
         lock_results: Vec<Result<()>>,
+        credit_only_locks: Vec<Result<CreditOnlyLocks>>,
         hash_queue: &BlockhashQueue,
         error_counters: &mut ErrorCounters,
-    ) -> Vec<Result<(InstructionAccounts, InstructionLoaders, InstructionCredits)>> {
+    ) -> Vec<
+        Result<(
+            InstructionAccounts,
+            InstructionLoaders,
+            InstructionCredits,
+            CreditOnlyLocks,
```

Comment:
Doesn't seem necessary to return this CreditOnlyLocks since its passed into the function as an argument and remains unchanged

Reply:
Agreed